src/components/EmbeddedWalletLayout/styles.scss

```
@@ -0,0 +1,134 @@
+.EmbeddedWalletLayout {
+  position: fixed;
+  inset: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: var(--sds-clr-gray-02);
+  box-sizing: border-box;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+.EmbeddedWalletHeader {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px;
+  height: 56px;
+  background: var(--sds-clr-gray-01);
+  border-bottom: 1px solid var(--sds-clr-gray-05);
+  z-index: 2;
+}
+
+.EmbeddedWalletHeader__brand {
+  display: flex;
+  align-items: center;
+  gap: 8px;
+  font-size: 16px;
+}
+
+.EmbeddedWalletHeader__brand img {
+  height: 24px;
+  width: auto;
+  object-fit: contain;
+}
+
+.EmbeddedWalletHeader__title {
+  color: var(--sds-clr-gray-11);
+  font-size: 12px;
+  font-weight: 500;
```

Let's use SDS variables where possible. You can see them by inspecting `root` in developer tools.

```suggestion
  font-weight: var(--sds-fw-medium);
```